lib/supavisor_web/endpoint.ex

```diff
@@ -10,6 +10,14 @@ defmodule SupavisorWeb.Endpoint do
     signing_salt: "zJOrGxcM"
   ]
 
+  socket "/socket/vercel", SupavisorWeb.WsProxy,
```

Comment:
Hmm on second thought maybe we keep "/vercel" out of this. Are there any other clients which implement the wire protocol over websockets?